### PR TITLE
LinkedIn.SearchAdCreatives

### PR DESCRIPTION
**Pull request by RikuVirtanen**
#7

### CODE REVIEW

File: Frends.LinkedIn.SearchAdCreatives/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Changelog
+
+## [1.0.0] - 2023-10-31
+### Changed
```
Comment (Svenskapojkarna) [inline Frends.LinkedIn.SearchAdCreatives/CHANGELOG.md]: `Added`.

File: Frends.LinkedIn.SearchAdCreatives/Frends.LinkedIn.SearchAdCreatives/LinkedIn.cs [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+    /// Main class of the Task.
+    /// </summary>
+    public static class LinkedIn
+    {
+        private static readonly IHttpClientFactory ClientFactory = new HttpClientFactory();
+        private static readonly ObjectCache ClientCache = MemoryCache.Default;
+        private static readonly CacheItemPolicy CachePolicy = new CacheItemPolicy() { SlidingExpiration = TimeSpan.FromHours(1) };
+
+        /// <summary>
+        /// This is Task.
```
Comment (Svenskapojkarna) [inline Frends.LinkedIn.SearchAdCreatives/Frends.LinkedIn.SearchAdCreatives/LinkedIn.cs]: Please update documentation.

File: Frends.LinkedIn.SearchAdCreatives/Frends.LinkedIn.SearchAdCreatives/LinkedIn.cs [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+        /// [Documentation](https://tasks.frends.com/tasks/frends-tasks/Frends.LinkedIn.SearchAdCreatives).
+        /// </summary>
+        /// <param name="filter">Filter parameters.</param>
+        /// <param name="options">Options parameters.</param>
+        /// <param name="cancellationToken">Cancellation token given by Frends.</param>
+        /// <returns>Object { string Output }.</returns>
```
Comment (Svenskapojkarna) [inline Frends.LinkedIn.SearchAdCreatives/Frends.LinkedIn.SearchAdCreatives/LinkedIn.cs]: Copy-paste error.

File: Frends.LinkedIn.SearchAdCreatives/Frends.LinkedIn.SearchAdCreatives/LinkedIn.cs [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+    using System.ComponentModel;
+    using System.Linq;
+    using System.Net;
+    using System.Net.Http;
+    using System.Net.Http.Headers;
+    using System.Runtime.Caching;
+    using System.Text;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using System.Web;
```
Comment (Svenskapojkarna) [inline Frends.LinkedIn.SearchAdCreatives/Frends.LinkedIn.SearchAdCreatives/LinkedIn.cs:14]: This namespace is not used, so it can be removed.

Comment (RikuVirtanen) [reply Frends.LinkedIn.SearchAdCreatives/Frends.LinkedIn.SearchAdCreatives/LinkedIn.cs:14]: The namespace is used with HttpUtility class

**Pull request merged**